torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -893,13 +902,13 @@ RegisterOperators reg(
      // results
      DEFINE_GENERIC_OP_WITH_COMPLEX(
          aten::pow,
-         static_cast<double>(pow(a, b)),
-         static_cast<double>(pow(a, b)),
+         powWrapper(a, b),
+         powWrapper(a, b),
          static_cast<c10::complex<double>>(pow(a, b)),
          float,
          float,
          complex),
-     DEFINE_INT_FLOAT_OP(aten::pow, pow(a, b), float),
+     DEFINE_INT_FLOAT_OP(aten::pow, powWrapper(a, b), float),
      DEFINE_FLOAT_COMPLEX_OP(aten::pow, pow(a, b), complex),
      DEFINE_SCALAR_BINARY_OP_AVOID_COLLISION(
```

Comment:
I am not sure how to trigger the `DEFINE_SCALAR_BINARY_OP_AVOID_COLLISION` from Python. What Python type correspond to a scalar?

Reply:
same, I don't know how to test this right now either, but since `Scalar` is just union of all cases you've tested, it won't be an issue to just replace all `pow` calls.